README.md

```diff
@@ -0,0 +1,142 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://user-images.githubusercontent.com/65084923/236905326-efe684f3-a1d1-4acb-af4d-cf290f8b3982.png" alt="logo" width="100"  height="auto" />
+  <img src="https://user-images.githubusercontent.com/65084923/232604539-ea646b45-13a1-4214-9ffa-6720ec9634bf.png" alt="logo" width="100"  height="auto" />
+  <br/>
+  <img src="https://user-images.githubusercontent.com/65084923/220200035-01fa5e08-af6a-495b-b597-55157a2b7d95.gif" alt="logo" />
+  <br/>
+
+  <h3><b>Blog App - Ruby on Rails</b></h3>
+  <h3><b>Blog App - Ruby on Rails - ROR Microverse Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+
+- [💻 Getting Started](#getting-started)
+  - [Available Scripts](#available-scripts)
+  - [Clone](#clone)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Blog App - Ruby on Rails - ROR Microverse Project<a name="about-project"></a>
+
+> Blog App - developed with Ruby on Rails.
```

Comment:
- [ ] [optional] It would have been nice to add a more details description of your project, maybe what the app is about and its main futures in summary.